checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -27,3 +27,25 @@ class Config {
         return [Config]::new($use_rcc)
     }
 }
+function CreateCommand([Config]$config) {
```

Comment:
Return type (if possible)?

Reply:
The return type can't be directly defined, but it is "hinted" at the bottom with  `return [PSCustomObject]`.
I can make it more explicit and create a class `Command` that would have the executable and argumentsList and then return it. Does that sound OK?

Reply:
That's a good idea, let's do that :+1: